### PR TITLE
Added config and main file for Migration

### PR DESCRIPTION
**Pull request by RohanDudeja**
Added config and main file for Migration

### CODE REVIEW

File: config/database.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import (
+	"database/sql"
+	"fmt"
+	"github.com/kelseyhightower/envconfig"
+	_ "github.com/kelseyhightower/envconfig"
+	"gopkg.in/yaml.v2"
+	"os"
+)
+
+var DB *sql.DB
```
Comment (mukesh0513) [inline config/database.go]: Is this needed here?

File: config/database.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+// Config represents configuration
+type Config struct {
+	Database struct{
+		Host     string `yaml:"host", envconfig:"DB_HOST"`
```
Comment (mukesh0513) [inline config/database.go]: run gofmt.

Comment (mukesh0513) [reply config/database.go]: Can we create another struct for Database? You can better expand this for other configurations then.

File: cmd/migrate/main.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+var (
+	flags = flag.NewFlagSet("migrate", flag.ExitOnError)
+	dir   = flags.String("dir", "../../migration", "directory with migration files")
+)
+func main() {
+	flags.Usage = usage
+	flags.Parse(os.Args[1:])
+	args := flags.Args()
+
+	command := args[0]// command like up, down
```
Comment (mukesh0513) [inline cmd/migrate/main.go]: what if no argument passed?

File: config/database.go (hunk 1 above)
Comment (RohanDudeja) [reply config/database.go]: removed

File: config/database.go (hunk 2 above)
Comment (RohanDudeja) [reply config/database.go]: done

File: cmd/migrate/main.go (hunk 3 above)
Comment (RohanDudeja) [reply cmd/migrate/main.go]: done

**Review by mukesh0513 — APPROVED** [review]
LGTM

**Comment by mukesh0513** [comment]
Resolve conflicts.

**Pull request merged**